include/bluetooth/hci.h

```diff
@@ -1566,8 +1583,20 @@ struct bt_hci_evt_le_phy_update_complete {
 } __packed;
 
 #define BT_HCI_EVT_LE_EXT_ADVERTISING_REPORT    0x0d
+
+#define BT_HCI_LE_ADV_EVT_TYPE_CONN                 BIT(0)
+#define BT_HCI_LE_ADV_EVT_TYPE_SCAN                 BIT(1)
+#define BT_HCI_LE_ADV_EVT_TYPE_DIRECT               BIT(2)
+#define BT_HCI_LE_ADV_EVT_TYPE_HI_DC_CONN           BIT(3)
+#define BT_HCI_LE_ADV_EVT_TYPE_LEGACY               BIT(4)
+
+#define BT_HCI_LE_ADV_EVT_TYPE_DATA_STATUS(ev_type) (((ev_type) >> 5) & 0x03)
+#define BT_HCI_LE_ADV_EVT_TYPE_DATA_STATUS_COMPLETE   0
+#define BT_HCI_LE_ADV_EVT_TYPE_DATA_STATUS_PARTIAL    1
+#define BT_HCI_LE_ADV_EVT_TYPE_DATA_STATUS_INCOMPLETE 2
```

Comment:
`BT_HCI_LE_ADV_EVT_TYPE_DATA_STATUS_INCOMPLETE` is a bit misleading name, as the value 1 also represents an incomplete report. I would suggest the following:
 - `BT_HCI_LE_ADV_EVT_TYPE_DATA_STATUS_INCOMPLETE_MORE_DATA`
 - `BT_HCI_LE_ADV_EVT_TYPE_DATA_STATUS_INCOMPLETE_TRUNCATED`

Reply:
These are getting really long. Can we drop something without hurting clarify? Perhaps `TYPE`?